clients/admin-ui/src/features/datamap/datamap.slice.ts

```diff
@@ -102,6 +102,7 @@ const datamapApi = baseApi.injectEndpoints({
           url: `plus/datamap/minimal?${queryString}`,
         };
       },
+      providesTags: ["Datamap"],
```

Comment:
This is to allow the datamap report table to reload when there are changes to systems